src/util.ts

```diff
@@ -67,3 +68,16 @@ export const getTracesEndpoint = (options?: HoneycombOptions) => {
 export const getTracesApiKey = (options?: HoneycombOptions) => {
   return options?.tracesApiKey || options?.apiKey;
 };
+
+export const getSampleRate = (options?: HoneycombOptions) => {
+  if (
+    // sample rate must be a whole integer greater than 0
+    options?.sampleRate &&
+    options?.sampleRate > 0 &&
+    options?.sampleRate % 1 === 0
```

Comment:
there's also `Number.isInteger(value)` if you fancy — 

```
if (Number.isInteger(options?.sampleRate) && options.sampleRate > 0) {
  return options.sampleRate;
}
```

Reply:
Yeah I actually went back and forth with this haha. That seems like the more proper thing and in fact, `Number.isSafeInteger` is even better. When I had the 3 lines I didn't like the inconsistency but could use this anyway.

Reply:
https://github.com/honeycombio/honeycomb-opentelemetry-web/pull/70/commits/7a62e1286da5672acaca1395b9b514b7f5302366

Reply:
`Number.isSafeInteger(undefined)` returns false too — using `Number.isSafeInteger` is like a two-fer: one check for both undefined _and_ numbers that aren't integers in one compact little check :party-cat: